vue-app/src/views/TransactionSuccess.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div class="container">
+    <div class="image-wrapper">
+      <img class="docking" src="@/assets/docking.png" />
+      <img class="money" src="@/assets/money.gif" />
+      <img class="money" src="@/assets/confetti.gif" />
+    </div>
+    <div class="content">
+      <span class="contributed-icon">🎉</span>
+      <p
+        v-if="$route.params.type === 'reallocation'"
+        class="contributed-header"
+      >
+        {{ formatContribution() }}
+        {{ $store.state.currentRound.nativeTokenSymbol }} reallocated!
+      </p>
+      <p
+        v-else-if="$route.params.type === 'contribution'"
+        class="contributed-header"
+      >
+        You just contributed!
+      </p>
+      <div>
+        <p
+          v-if="$route.params.type === 'reallocation'"
+          class="contributed-content"
+        >
+          Your choices have been updated! You can update your choices again any
+          time in the next
+          <span class="contributed-content-bold"
+            >{{ reallocationTimeLeft.days }} days
+            {{ reallocationTimeLeft.hours }} hours</span
+          >.
+        </p>
+        <p
+          v-else-if="$route.params.type === 'contribution'"
+          class="contributed-content"
+        >
+          Thanks for contributing to the Eth2 ecosystem. If you change your
+          mind, you have
+          <span class="contributed-content-bold"
+            >{{ reallocationTimeLeft.days }} days
+            {{ reallocationTimeLeft.hours }} hours</span
+          >
+          to reallocate your contributions.
+        </p>
+        <div class="receipt">
+          <transaction-receipt :hash="`${$route.params.hash}`" />
+        </div>
+        <div class="input-button">
+          <button class="contributed-button" @click="redirectToProjects()">
+            <span>OK</span>
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { BigNumber } from 'ethers'
+import Vue from 'vue'
+import Component from 'vue-class-component'
+
+// API
+import { RoundInfo, TimeLeft } from '@/api/round'
+
+// Components
+import TransactionReceipt from '@/components/TransactionReceipt.vue'
+
+// Utils
+import { formatAmount } from '@/utils/amounts'
+import { getTimeLeft } from '@/utils/dates'
+
+@Component({
+  components: { TransactionReceipt },
+})
+export default class TransactionSuccess extends Vue {
+  get contribution(): BigNumber | null {
+    return this.$store.state.contribution
+  }
+
+  get currentRound(): RoundInfo | null {
+    return this.$store.state.currentRound
+  }
+
+  get reallocationTimeLeft(): TimeLeft | string {
```

Comment:
I would suggest trying to return always one data type. In this case, would be just `TimeLeft`. Also, looking at this line:
```
{{ reallocationTimeLeft.days }} days // line #42
```
returning a string would break it.